pos_customer_required/static/src/js/ProductScreen.js

```diff
@@ -0,0 +1,25 @@
+/* Copyright NuoBiT Solutions - Eric Antones <eantones@nuobit.com>
+   Copyright NuoBiT Solutions - Kilian Niubo <kniubo@nuobit.com>
+   License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl) */
+odoo.define("pos_customer_required.ProductScreen", function (require) {
+    "use strict";
+
+    const ProductScreen = require("point_of_sale.ProductScreen");
+    const Registries = require("point_of_sale.Registries");
+
+    var PCRProductScreen = (ProductScreen) =>
+        class PCRProductScreen extends ProductScreen {
```

Comment:
``        class extends ProductScreen {``
should be enought. (also in Payment screen).

(see other v14.0 module like (``pos_margin``, ``pos_access_right``)

Reply:
Done!
Thanks @legalsylvain